railties/lib/rails/generators/rails/app/templates/config/initializers/handle_cookies_without_metadata.rb.tt

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
+
+# Signed and encrypted cookies now have metadata associated with it.
+# If you don't want your existing cookies to be rejected immediately you need to specify a window
+# for how long those cookies should be honored as a Time or ActiveSupport::Duration object.
+
+# Also, setting this value to nil or to a date in the past rejects the existing cookies upfront.
+Rails.application.config.action_dispatch.fail_cookies_without_metadata_after = <%= ActiveSupport::JSON.encode 1.month.from_now %>
```

Comment:
Maybe https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_0.rb.tt is a better place for this?

@kaspth wdyt?

Reply:
@guilleiguaran Yes, that's the place we should put this.